### PR TITLE
Update urts.rs

### PR DESCRIPTION
**Pull request by goodengineer**
# Description of change

Please write a summary of your changes and why you made them. 

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Documentation Fix

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes
- [ ] I have updated the CHANGELOG.md, if my changes are significant enough

### CODE REVIEW

**Review by samuel-rufi — APPROVED** [review]
Great! This is better.

File: bee-tangle/src/urts.rs [hunk 1]
```diff
@@ -36,6 +36,8 @@ const MAX_AGE_SECONDS_AFTER_FIRST_CHILD: u8 = 3;
 // used to widen the cone of the tangle.
 const MAX_NUM_CHILDREN: u8 = 2;
 
+const OPTIMAL_NUM_TIPS: u8 = 4;
```
Comment (thibault-martinez) [inline bee-tangle/src/urts.rs]: Doesn't compile, this needs to be a `usize`.

**Review by thibault-martinez — CHANGES_REQUESTED** [review]

File: bee-tangle/src/urts.rs (hunk 1 above)
Comment (goodengineer) [reply bee-tangle/src/urts.rs]: Tnx, replaced u8 to usize, what now?

File: bee-tangle/src/urts.rs [hunk 2]
```diff
@@ -36,6 +36,8 @@ const MAX_AGE_SECONDS_AFTER_FIRST_CHILD: u8 = 3;
 // used to widen the cone of the tangle.
 const MAX_NUM_CHILDREN: u8 = 2;
```
Comment (thibault-martinez) [inline bee-tangle/src/urts.rs:38]:
```suggestion
```

**Comment by thibault-martinez** [comment]
This has been moved to the dev branch, please open the PR again on the dev branch if you want.

**Pull request closed without merging**